python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/port-channel-interfaces.j2

```diff
@@ -15,30 +15,52 @@
 | Interface | Description | Mode | VLANs | Native VLAN | Trunk Group | LACP Fallback Timeout | LACP Fallback Mode | MLAG ID | EVPN ESI |
 | --------- | ----------- | ---- | ----- | ----------- | ------------| --------------------- | ------------------ | ------- | -------- |
 {%     for port_channel_interface in port_channel_interfaces | arista.avd.natural_sort('name') %}
-{%         if port_channel_interface.type is arista.avd.defined("switched") %}
+{#  Below if condition is added to keep the existing support for rendering switchport attributes on the basis of port_channel_interface.type == switched #}
+{#  TODO: AVD6.0 this if condition should be removed once the port_channel_interface.type key is removed in AVD6.0 #}
+{%         if (port_channel_interface.switchport.mode is arista.avd.defined or
+            port_channel_interface.switchport.access_vlan is arista.avd.defined or
+            port_channel_interface.switchport.trunk.allowed_vlan is arista.avd.defined or
+            port_channel_interface.switchport.trunk.native_vlan_tag is arista.avd.defined(true) or
+            port_channel_interface.switchport.trunk.native_vlan is arista.avd.defined or
+            port_channel_interface.switchport.trunk.groups is arista.avd.defined) and
+            (port_channel_interface.type is not arista.avd.defined or (port_channel_interface.type is arista.avd.defined and port_channel_interface.type not in ["switched", "routed"])) %}
 {%             set description = port_channel_interface.description | arista.avd.default("-") %}
-{%             set mode = port_channel_interface.mode | arista.avd.default("access") %}
-{%             set vlans = port_channel_interface.vlans | arista.avd.default("-") %}
-{%             if  port_channel_interface.native_vlan_tag is arista.avd.defined(true) %}
+{%             set mode = port_channel_interface.switchport.mode | arista.avd.default('-') %}
+{%             set vlans = [] %}
+{%             if port_channel_interface.switchport.access_vlan is arista.avd.defined %}
+{%                 do vlans.append(port_channel_interface.switchport.access_vlan) %}
+{%             endif %}
+{%             if port_channel_interface.switchport.trunk.allowed_vlan is arista.avd.defined %}
+{%                 do vlans.extend(port_channel_interface.switchport.trunk.allowed_vlan | arista.avd.range_expand | map('int')) %}
+{%             endif %}
+{%             set vlans = vlans | arista.avd.list_compress %}
```

Comment:
It is creating empty column in documentation. Adding default `-` for vlans will help.

Reply:
Fixed